cafebabel/core/commands.py

```diff
@@ -1,31 +1,24 @@
 import click
-from peewee import create_model_tables, drop_model_tables
 
-from .. import app, db
-from ..users.models import Role, User, UserProfile, UserRoles, user_datastore
-
-tables = [User, UserProfile, Role, UserRoles]
+from .. import app
+from ..users.models import Role, User, UserProfile, user_datastore
 
 
 def _initdb():
     _dropdb()
-    create_model_tables(tables, fail_silently=True)
-    Role.create(name='editor')
-    admin_role = Role.create(name='admin')
+    Role(name='editor').save()
```

Comment:
Note you can also do `Role.create(name='editor')`

Reply:
It doesn't look to be the case, source?

Reply:
Meant `Role.objects.create(name='editor')`
http://docs.mongoengine.org/apireference.html#mongoengine.queryset.QuerySet.create

Reply:
It makes more sense to me to have an EmbeddeDocument as:

- it is fully dependant of the `User` model and should therefore not be a autonomous `Document`
- it has a schema to respect and should therefore not be a json or object field